versioned_docs/version-v2.18.x/user-guide/configure-zowe-zosmf-workflow-2-18.md

```diff
@@ -0,0 +1,166 @@
+# Configuring Zowe with z/OSMF Workflows
+
+After you install Zowe, you can register and execute the z/OSMF workflows in the web interface to perform a range of
+Zowe configuration tasks. z/OSMF helps to simplify the Zowe configuration tasks and does not require the level of
+expertise that is needed to perform manual Zowe configuration. This configuration method also runs the `zwe init`
+command to initialize Zowe z/OS runtime.
+
+:::info Required role: system programmer
+:::
+
+## Prerequisites
+
+Ensure that you meet the following requirements before you start your Zowe configuration:
+
+- Install and configure z/OSMF
+- Install Zowe with an SMP/E build, PSWI, or a convenience build
+
+## Overview of Full Zowe server-side configuration for Zowe 2.0 workflow
+
+You can complete the following tasks with the **Full Zowe server-side configuration for Zowe 2.0** workflow:
+
+- Configure the Zowe instance
+- Enable the Gateway service
+- Enable the API Catalog
+- Enable the Discovery service
+- Enable the Caching service
+- Enable the Cloud gateway service
+- Enable the Metrics service
+- Enable the Application server
+- Enable ZSS component
+- Enable the jobs API
+- Enable the files API
+- Enable JES Explorer
+- Enable MVS Explorer
+- Enable USS Explorer
+
+You can execute the **Full Zowe server-side configuration for Zowe 2.0** workflow either from a PSWI during deployment or later from a created software
+instance in z/OSMF. Alternatively, you can execute the configuration workflow in z/OSMF during the workflow registration process.
+
+The configuration workflow described in this article is executed directly from a deployment of the Zowe PSWI.
+
+## Executing Full Zowe server-side configuration workflow for Zowe 2.0 from PSWI
+
+After Zowe installation, follow this procedure to perform Zowe server-side configuration.
+
+:::tip
+For more information, see [Installing Product Software Using z/OSMF Deployments](./install-zowe-pswi-deployment.md).
+:::
+
+1. In the **Software Management** panel, click **Actions** and **Add from z/OSMF System**. 
+2. Specify your system name and file location path (UNIX file).
+3. Click **Retrieve**. 
+4. Review the Deployment Jobs and click **Deployment Checklist**.
+5. Follow each step outlined in the **Deployment Checklist**.
+
+![Deployment Checklist](../images/zosmf/perform-workflows.png)
+
+As presented in the **Deployment Checklist**, during deployment you have the option to perform workflows.
+
+6. After you complete all of the preceding steps outlined in the Deployment checklist, click **Perform workflows** to leverage configuration with z/OSMF workflows.
+
+    Each workflow appears in the panel. 
+
+:::note
+The **Perform Workflows** step enables you to run either all attached workflows or just the
+    post-deployment workflow for mounting, which is required.
+:::
+
+:::caution Important
+Ensure that you first perform the **Zowe Mount** workflow as described in the IBM documentation.
```

Comment:
Please provide the name of the IBM documentation that describes performing the Zowe Mount point. We do not need to provide the link in IBM doc, but we should provide the article name.

Reply:
I would say, the best what IBM provides and is relevant (because workflow from PSWI execution is slightly different from native execution from workflow tab) are those 2 links https://www.ibm.com/docs/en/zos/3.1.0?topic=overview-about-workflows  https://www.ibm.com/docs/en/zos/3.1.0?topic=task-managing-steps-in-workflow

Reply:
and what you are interested in is this, session from GSE about PSWI deployments, namely slide 62 and following, https://www.newera.com/INFO/061422_zOSMF_SM.pdf , and in previous slides you see the screen over which user have to jump before he get to phase where he can perform workflows within deployment/install phase